src/shared/tpm2-util.c

```diff
@@ -480,6 +480,56 @@ void tpm2_pcr_mask_to_selection(uint32_t mask, uint16_t bank, TPML_PCR_SELECTION
         };
 }
 
+static void tpm2_log_debug_buffer(const void *buffer, const size_t size, const char *msg) {
```

Comment:
Please drop `const` for `size`. We usually set `const` for arguments only for pointers.

Reply:
sorry i should have caught that, will drop it, thanks